backend/bruno/KSummarized/ToDo/Items/Delete.bru

```diff
@@ -0,0 +1,46 @@
+meta {
+  name: Delete
+  type: http
+  seq: 5
+}
+
+delete {
+  url: https://{{api_base_url}}/api/todo/items/{{itemId}}
+  body: none
+  auth: bearer
+}
+
+auth:bearer {
+  token: {{token}}
+}
+
+body:json {
```

Comment:
Why in the `Delete` method which is the `no-body` you have a body? Is it required for anything?

Reply:
I have copied the requests from the Create one, I have not noticed that even setting to `no-body` in UI it does not delete it in the file